packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Provide more legible errors to user when they are missing user/password credentials while
+running `zowe zos-uss issue ssh` commands. [#2081](https://github.com/zowe/zowe-cli/pull/2081)
```

Comment:
Please update the changelog since error formatting wasn't changed in this PR. The fix for CLI is to prompt for user/password on SSH commands when a token is stored in the config.